packages/core/utils/src/totals/line-item/index.ts

```diff
@@ -66,6 +70,40 @@ export function getLineItemsTotals(
   return itemsTotals
 }
 
+function setRefundableTotal(
+  item: GetItemTotalInput,
+  discountTotal: BigNumberInput,
+  totals: GetItemTotalOutput,
+  context: GetLineItemsTotalsContext
+) {
+  const totalReturned = MathBN.sum(
+    item.return_requested_quantity ?? 0,
```

Comment:
question: should we only consider `return_received_quantity` as a returned quantity

Reply:
That quantity decreases when the the item is actually received or dismissed.

Reply:
Ok got it!